.circleci/config.yml

```diff
@@ -194,6 +194,21 @@ jobs:
             rm -rf /home/circleci/.gnupg
             rm -rf /home/circleci/.npmrc
 
+  update-cm:
+    executor: cif_executor
+    working_directory: ~/repo
+    steps:
+      - checkout
+      - run:
+          name: Update CM Repository
+          command: |
+            git config credential.helper '!f() { sleep 1; echo "username=${GIT_USER}"; echo "password=${GIT_PASSWORD}"; }; f'
+            git remote add downstream $GIT_REPO
+            git fetch downstream
+            git checkout -b tmp downstream/main
+            git merge $CIRCLE_TAG
```

Comment:
What happens in case there is a conflict? I suspect that `git merge` will simply not create a commit, and the push doesn't do anything. Could we fail this job in this case, so we get notified that someone has to manually resolve the conflict.

Reply:
I assume that is what happens already. `git merge` will `exit 1` if the merge fails. This should cause the job to fail and we will be notified. However it should not fail the other steps. 

It would be good to have a tri-state including "unstable" but afaics there is nothing like that for circleci.